lib/requests/password-reset-email-request.js

```diff
@@ -123,7 +130,9 @@ class PasswordResetEmailRequest extends AuthRequest {
     return this.accountManager.accountExists(username)
       .then(exists => {
         if (!exists) {
-          throw new Error('Account not found for that username')
+          const error = new Error("Account not found for that username")
+          error.code = "ACCOUNT_MISSING"
+          throw error
```

Comment:
```suggestion
          // For security avoid using an error that leak information
          return renderSuccess()
```
`renderSuccess()` is may be changed to better name like resetLinkMessage()
The message itself should be changed in 
https://github.com/nodeSolidServer/node-solid-server/blob/d8290cbe3f7fe5ba7ccd46e988a1591659c6de93/default-views/auth/reset-link-sent.hbs#L17

Reply:
I changed the wording and renamed it to resetLinkMessage in the latest commits